README.md

```diff
@@ -109,7 +122,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+FSM.pdf
```

Comment:
# 4

I'm curious how you arrived at this FSM based on the BNF. 

There's a problem with it, though. You're in S0, and you receive the word "the". Where do you go: S1 or S2? In your diagram it could be either, which makes your state machine nondeterministic.

What should happen is exactly what you wrote in the answer to the next question.